tests/integration/telemetry/tracing/clienttracing/client_tracing_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019 Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clienttracing
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	uuid "github.com/satori/go.uuid"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/util/retry"
+	util "istio.io/istio/tests/integration/mixer"
+	"istio.io/istio/tests/integration/telemetry/tracing"
+)
+
+var (
+	traceHeader = "X-Client-Trace-Id"
```

Comment:
nit: use all lowercase (to avoid confusion over header case handling)

Reply:
done